features/timeseries/src/test/java/org/opennms/netmgt/timeseries/samplewrite/TimeseriesRoundtripIT.java

```diff
@@ -149,15 +150,16 @@ public void setUp() {
         config.put(CONFIG_KEY_FOR_CATEGORIES, "true");
         config.put(CONFIG_PREFIX_FOR_TAGS + "resource_label", "${resource:label}");
         config.put(CONFIG_PREFIX_FOR_TAGS + "node_location", "${resource:location}");
-        config.put(CONFIG_PREFIX_FOR_TAGS + "node_label", "${resource:node_label}");
```

Comment:
`node_label` should still be there, right?

Reply:
It's already there at line 146, so removed it.